plugins/commands/global-status/command.rb

```diff
@@ -65,6 +65,17 @@ def execute
           @env.machine_index.delete(deletable) if deletable
         end
 
+        # Machine-readable (non-formatted) output
+        @env.ui.machine("metadata", "machine-count", entries.length.to_s);
+        entries.each do |entry|
+          opts = { "target" => entry.send(:name).to_s }
+          @env.ui.machine("machine-id", entry.send(:id).to_s[0...7], opts)
+          @env.ui.machine("provider-name", entry.send(:provider).to_s, opts)
+          @env.ui.machine("machine-home", entry.send(:vagrantfile_path).to_s, opts)
+          @env.ui.machine("state", entry.send(:state).to_s, opts)
+        end
```

Comment:
The `#send` method isn't required here. It is useful for accessing instance methods which may not be publicly accessible (are marked as `protected` or `private`) or dynamically calling a method (like when the method name is defined within a variable). Instead the methods can just be called directly: `entry.name`.

Reply:
Removed the unnecessary send() calls.